src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -5,17 +5,18 @@
     <BuildScript>$(ProjectDirectory)eng\build$(ShellExtension)</BuildScript>
 
     <!-- On Windows, build all for the VB PoC -->
-    <BuildActions Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(BuildOS)' == 'windows'">$(FlagParameterPrefix)restore $(FlagParameterPrefix)all $(FlagParameterPrefix)pack</BuildActions>
+    <BuildActions Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(BuildOS)' == 'windows'">$(FlagParameterPrefix)restore $(FlagParameterPrefix)all $(FlagParameterPrefix)pack -NativeToolsOnMachine</BuildActions>
 
     <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)arch $(TargetArchitecture)</BuildArgs>
-    <BuildArgs Condition="'$(DotNetBuildSourceOnly)' == 'true' or '$(BuildOS)' != 'windows'">$(BuildArgs) $(FlagParameterPrefix)no-build-repo-tasks</BuildArgs>
+    <BuildWithDesktopMSBuild>true</BuildWithDesktopMSBuild>
```

Comment:
What's the reason for that? Is this temporary?

Reply:
ASPNet uses desktop msbuild for some of the build, specifically the native code (via vcxproj). It uses core msbuild for the rest as much as possible. This switch property avoids passing `-msbuildEngine dotnet` to the build invocation, which is unsupported in the aspnet build scripts.